src/components/common/FileUploadPreview.vue

```diff
@@ -68,23 +71,20 @@ export default class FileUploadPreview extends Mixins(DocumentMixin) {
   async mounted (): Promise<void> {
     if (this.inputFile) {
       this.fileUpload = this.inputFile
+      // wait for v-file-input to settle
       await this.$nextTick()
-      this.isFileValid(this.fileUpload)
     }
-  }
-
-  async fileChange (file): Promise<void> {
-    await this.emitFileSelected(file)
```

Comment:
This is was only called by the template so I combined it into emitFileSelected() (and then renamed it). This is minor cleanup, not a refactor.